packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -447,3 +450,30 @@ export async function updateDenormalizedContributorsList(tag: DbTag): Promise<Co
   
   return contributionStats;
 }
+
+defineQuery({
+  name: "UserTopTags",
+  resultType: "[TagWithCommentCount!]",
+  argTypes: "(userId: String!)",
+  schema: `
+    type TagWithCommentCount {
+      tag: Tag
+      commentCount: Int!
+    }`,
+  fn: async (root: void, {userId}: {userId: string}, context: ResolverContext) => {
+    const tagsRepo = new TagsRepo();
```

Comment:
It's not a big problem but just as a note for the future: when you create a new repo you should add it to `packages/lesswrong/server/repos/index.ts` then it will be available for all requests as `context.repos.tags`. It's very rare that you should ever actually need to do `new XRepo()` (basically, only in places where there's no context available).